sandpack-client/README.md

```diff
@@ -257,6 +257,19 @@ worker externally.
 > I want to highlight that you can also host the bundler by yourself, all
 > necessary files are in the `sandpack` folder.
 
+### Self Hosting Bundler
+
+If you want to host the bundler yourself, you will need to do a few things.
+
+* The bundler is part of the codesandbox-client codebase: https://github.com/codesandbox/codesandbox-client
+* Clone the codesandbox-client and install the dependencies in the root folder (`yarn install`).
+* `yarn build:deps` to build some of the packages lerna needs for internal links.
+* create your instance of sandpack with `yarn build:sandpack`.
+
+*This creates a www folder in the root folder that www folder is the sandpack folder that sandpack-client connects to on {version}-sandpack.codesandbox.io
+once you have this hosted on your end, you can pass bundlerURL when calling new SandpackClient(iframe, sandboxInfo, { bundlerURL: 'https://your-hosted-version' } )
```

Comment:
can you try adding the code snippet in a code block formatted between triple-backticks? It becomes more readable. also, you should format this with prettier if it's possible, it will rearrange the readme content to a max of 80 characters per line I think. don't have prettier on pre-commit yet, but there's a task for that on the issues list